docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -454,6 +454,68 @@ A feature store consists of five main components:
 
     Feature Store Component Hierarchy.
 
+Connecting from Amazon SageMaker
+------------------------------------
```

Comment:
@steffengr length needs to match title length

Reply:
Fixed